### PR TITLE
Staffbranch

### PR DESCRIPTION
**Pull request by KartikaNaveena**

### CODE REVIEW

File: Hotel/forms.py [hunk 1]
```diff
@@ -9,4 +9,17 @@ class RegisterForm(UserCreationForm):
...
+class BookingForm(forms.Form):
+    room_type = forms.ModelChoiceField(queryset=RoomType.objects.all())
+    check_in = forms.DateField()
+    check_out = forms.DateField()
+
+class RoomForm(forms.ModelForm):
+    class Meta:
+        model = Rooms
+        fields = ['number', 'room_type', 'price','capacity', 'status']
+class RoomaddForm(forms.ModelForm):
```
Comment (akshita31) [inline Hotel/forms.py:22]: Why this one ?

File: Hotel/models.py [hunk 2]
```diff
@@ -18,39 +19,32 @@ class Hotels(models.Model):
...
         return self.name
 
-
+class RoomType(models.Model):
+    name = models.CharField(max_length=100)
+    description=models.CharField(max_length=200,null=True)
+    def __str__(self):
+        return self.name
 class Rooms(models.Model):
     ROOM_STATUS = (
```
Comment (akshita31) [inline Hotel/models.py:28]: You can use an enum for the room status.

https://docs.python.org/3/library/enum.html

File: Hotel/forms.py (hunk 1 above)
Comment (KartikaNaveena) [reply Hotel/forms.py:22]: roodaddform is for adding of new room and roomform is for editing of already existing room . roomaddform contains all attributes .whereas in roomform hotel attribute is not present as it is not  to change hotel associated with the room as edit feature of room

File: Hotel/models.py (hunk 2 above)
Comment (KartikaNaveena) [reply Hotel/models.py:28]: Ok didi will check it

**Review by akshita31 — APPROVED** [review]

**Pull request merged**